ONSdigital/eq-questionnaire-validator
# Update max_number and min_number values

### What is the context of this PR?
Update max and min number values for validator. There should be 14 negative digits (12 whole numbers and 2 decimal numbers) and 15 positive digits (13 whole numbers and 2 decimal numbers).

There is a test needed for `validate_value_in_limits` But will be added in another Pull Request.
UPDATE: 
- Now also checks any reference values using answer sources (number validation)
- Made `def _get_numeric_value_for_value_source` into a external function.
### How to review

- Run functional tests
- Check if the changes made fit the [trello card](https://trello.com/c/0yJIjinU/5658-enable-up-to-15-digits-in-relevant-fields-1-sp)
- Run validator against a instance of runner in the `update-max-length-to-15-digits` branch
- All relevant tests are edited / made? 
### Checklist
